plugins/workspace-plugin/src/askpass-main.ts

```diff
@@ -0,0 +1,72 @@
+// tslint:disable:file-header
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+// Based on: https://github.com/che-incubator/vscode-git/blob/master/src/askpass-main.ts.
```

Comment:
May be we should use a link to original microsoft repository?

Reply:
I think it does make sense. I've changed references. Do you know if we need to create some CQ for that?

Reply:
**AFAIK** we do need a CQ for code pieces.